### DISCUSSION TITLE
Nativescript plugins repo?

### DISCUSSION THREAD

**Discussion by insytes** [triniwiz/nativescript-plugins#199] [General]
Are there any plans to combine this repo with [NativeScript/plugins](https://github.com/NativeScript/plugins)?

**Comment by triniwiz** [comment]
No I didn't have any reason to.

**Reply by insytes** [reply]
There are some great plugins in both repos. From a users perspective it would be great if all were under one roof

**Reply by insytes** [reply]
@triniwiz BTW accept this lousy [PR](https://github.com/triniwiz/nativescript-plugins/pull/197) 😄

**Reply by triniwiz** [reply]
Understood but I don't want to put too much burden on the team with my mess 😆

**Reply by NathanWalker** [reply]
It's a good question @insytes technically we could put them all in NativeScript/plugins - we've generally tried to keep more private business SDK related plugins in separate repos since they tend to change/evolve more organically than standard baseline helper plugins to standard platform features (NativeScript/plugins, more fundamental stuff) - that's at least the thought process.

**Reply by insytes** [reply]
@NathanWalker understood. I would say that the reasoning is sound except in my case of using the `videorecorder`. I made a placeholder issue on the NativeScript/plugins repo without realising my mistake.

**Comment by insytes** [comment]
> Understood but I don't want to put too much burden on the team with my mess 😆

Thanks for accepting @triniwiz 🚀

**Reply by insytes** [reply]
@triniwiz is there a release schedule for these packages? I'd like to pull the latest changes to the videorecorder

**Comment by triniwiz** [comment]
Soon, I originally planned to do some releases last weekend but worked on something else
